README.md

```diff
@@ -71,6 +71,17 @@ During a sprint we track progress using a [scrum or kanban-like board](#notes)<s
 
 After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs. 
 
+##Residencies
+
+In order to promote personal growth and inter-team knowledge transfer, the Mobile & TV Group offers residencies. A residency is a sprint-long period where anyone from any other group within Vimeo can join the team to work on any product that interests them. This allows the opportunity to learn new langagues and development skills.
+
+Residencies last for one sprint at a miniumum as to conform to the team process and sprint lifecycle. If you're doing a residency, you'll attend the sprint planning meetings and retrospective. You'll be paired up with a developer on the team to help get set up and provide guidance along the way.
```

Comment:
Consider removing the first sentence in this paragraph since it's redundant against the info in the preceding paragraph. 

Consider revising the second sentence to get a little more detail in there. Something like, "Residents" attend all sprint-related meetings, including A, B, and C. And their tasks are tracked in Jira alongside everyone else's. 

Consider revising the third sentence so it's not just developer-focused, but a QA or product manager or designer could participate. Consider adding a term like, coach, mentor, point-person or something to make it clear that you've got a partner there with you every step of the way.